common/lib/xmodule/xmodule/modulestore/courseware_index.py

```diff
@@ -129,7 +129,7 @@ def remove_index_item_location(item_location):
             error_list.append(_('General indexing error occurred'))
 
         if raise_on_error and error_list:
-            raise SearchIndexingError(_('Error(s) present during indexing'), error_list)
+            raise SearchIndexingError(_('Errors present during indexing'), error_list)
```

Comment:
This is an error message, which I suspect goes to logs, no? Should this really be i18n'ed?